tests/pagination/test_serializer.py

```diff
@@ -0,0 +1,51 @@
+import dataclasses
+import pytest
+from mock import Mock
+from rest_framework import serializers
+
+from winter.pagination import Page
+from winter.pagination import PagePosition
+from winter.pagination import PageSerializer
+
+
+@dataclasses.dataclass
+class _TestEntity:
+    number: int
+
+
+class _TestSerializer(serializers.Serializer):
+    number = serializers.IntegerField()
+
+@pytest.mark.parametrize(('limit', 'offset', 'expected_previous', 'expected_next'), (
+    (1, 1, 'test-url.com?limit=1', 'test-url.com?limit=1&offset=2'),
+    (None, None, None, None),
+    (2, 3, 'test-url.com?limit=2&offset=1', 'test-url.com?limit=2&offset=5'),
+    (9, 3, 'test-url.com?limit=9', None),
+))
+def test_page_serializer(limit, offset, expected_previous, expected_next):
+    TestPageSerializer = PageSerializer[_TestSerializer]
+    base_url = Mock()
+    base_url.return_value = 'test-url.com'
+    request = Mock()
+    request.build_absolute_uri = base_url
```

Comment:
You can replace it with `request.build_absolute_uri.return_value = 'test-url.com'`

Reply:
Done